test/e2e/demo.yaml

```diff
@@ -39,6 +39,12 @@ spec:
         args: ["$(AGENT_OPTS)","-jar","/app.jar"]
         ports:
           - containerPort: 8085
+        readinessProbe:
+          httpGet:
+            path: /hello
+            port: 8085
+          initialDelaySeconds: 3
+          periodSeconds: 3
```

Comment:
Could you add `failureThreshold: 10` to avoid some potential failures.

Reply:
get it.